samples/algorithms/sudoku-grover/Sudoku.qs

```diff
@@ -143,21 +137,102 @@ namespace Microsoft.Quantum.Samples.SudokuGrover {
         }
     }
 
+
+    /// # Summary
+    /// Encodes stating number constraints into amplitudes.
+    ///
+    /// # Inputs
+    /// ## nVertices
+    /// The number of vertices in the graph.
+    /// ## bitsPerColor
+    /// The bit width for number of colors.
+    /// ## startingNumberConstraints
+    /// The array of (Vertex#, Color) specifying the disallowed colors for vertices.
+    ///
+    /// # Output
+    /// A 2D array of amplitudes where the first index is the cell and the second index is the value of a basis state (i.e., value) for the cell
```

Comment:
It also makes sense to mention what are the values of the array elements - 1.0 for allowed values and 0.0 for prohibited ones